src/test/FreeC/PipelineTests.hs

```diff
@@ -24,3 +25,4 @@ testPipeline = do
   testPartialityAnalysisPass
   testResolverPass
   testTypeInferencePass
+  testExportPass
```

Comment:
I'd put this after `testEtaConversionPass` to keep the same order as in the imports.